forms/forms-view/src/main/java/org/bonitasoft/forms/client/view/controller/FormPagesViewController.java

```diff
@@ -216,6 +218,18 @@
     protected Map<String, Object> urlContext;
 
     /**
+     * Widget type to focused when loading is hidden
+     *
+     */
+    protected List<WidgetType> widgetTypesToFocus = Arrays.asList(
+            WidgetType.TEXT,
+            WidgetType.TEXTAREA,
+            WidgetType.TEXTBOX,
+            WidgetType.SUGGESTBOX,
+            WidgetType.RICH_TEXTAREA,
+            WidgetType.SUGGESTBOX_ASYNC);
+
```

Comment:
Shouldn't we add DATE, FILEUPLOAD, PASSWORD ?
And are we sure ther is no problem with CHECKBOX, CHECKBOX_GROUP, RADIOBUTTON_GROUP, LISTBOX_SIMPLE, LISTBOX_MULTIPLE ?